test/interchain/go.mod

```diff
@@ -92,7 +92,7 @@ require (
 	github.com/gtank/merlin v0.1.1 // indirect
 	github.com/gtank/ristretto255 v0.1.2 // indirect
 	github.com/hashicorp/go-cleanhttp v0.5.2 // indirect
-	github.com/hashicorp/go-getter v1.7.4 // indirect
```

Comment:
Do we really need to bump an indirect dependency?

Reply:
I don't see why not. It resolves a dependabot alert so why would we ignore it? https://github.com/celestiaorg/celestia-app/security/dependabot/38

Reply:
For this specific dependency, I agree that integrating it should not cause any issues as it does not introduce breaking changes. However, I would expect indirect dependencies to be updated by the modules that import them directly to ensure compatibility is preserved.